storybook-static/6.5e45ead7.iframe.bundle.js.LICENSE.txt

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
**nitpick:** Let's remove all bundled files (whole `storybook-static` directory). These files are present because you ran `yarn build-storybook`. We should not keep such files in repository. Such bundled files are used when we want to run Storybook on some server